include/simdjson/padded_string.h

```diff
@@ -17,6 +17,10 @@
 typedef int errno_t ;
 #endif // _MSC_VER
 
+#ifndef SIMDJSON_PADDING
```

Comment:
SIMDJSON_PADDING is defined in common_defs.h. Instead of redefinining it again, why not just do

#include "simdjson/common_defs.h"

?

Reply:
ok will remove in next PR